src/utils.sh

```diff
@@ -115,9 +115,5 @@ __enhancd::utils::nl()
 # __enhancd::utils::awk returns gawk if found, else awk
 __enhancd::utils::awk()
 {
-    if [[ "$(uname -s)" == "FreeBSD" ]]; then
-        [[ $(command -v gawk) ]] && echo "gawk" || echo "awk"
-    else
-        echo "awk"
-    fi
+    which gawk > /dev/null 2>&1 && echo "gawk" || echo "awk"
```

Comment:
Could you modify like the following representation?

```suggestion
    type gawk &>/dev/null && echo "gawk" || echo "awk"
```

I'd like to prefer `type` to `which` because it's faster and builtin command.